clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapAPI.java

```diff
@@ -1005,6 +1005,10 @@ private static CleverTapAPI fromBundle(final Context context, final Bundle extra
         return fromAccountId(context, _accountId);
     }
 
+    private static boolean isDevelopmentMode(Context context) {
+        return 0 != (context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE);
```

Comment:
make sure null checks are not needed here.

Reply:
I have added null checks to the invoking method, although it should probably not be needed. This is the context that is provided when creating the CleverTapAPI instance